index.html

```diff
@@ -6,11 +6,211 @@
     <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="style.css" />
 </head>
 
 <body>
-    <!-- Add your markup here -->
+    <section class="homepage-container">
+        <section class="hero-box">
+                <section class="hero-flexbox">
+                    <img src="img/AsterixetObelix.jpg"  class="AsterixetObelix" alt="Obelix holding cakes with Asterix standing next to him">
+
+                    <section class="menu-quote-container">
+
+<!-- 
+For the hamburger menu button I have chosen to use the functionality found at https://code-boxx.com/simple-responsive-pure-css-hamburger-menu/
+-->
+
+                    <!--
+                        <figure>
+                            <img
+                                src="img/menu-hamburger.svg"
+                                alt="Menu Button"
+                        </figure>
+                    --> 
+
+                        <section>
+                        <!-- (A) MENU WRAPPER -->
+                            <nav class="hamnav">
+                        
+                                <!-- (B) THE HAMBURGER -->
+    
+                                <!-- &#9776; is the HTML symbol for the “3 lines hamburger icon” -->
+                                    <span class="menugap"></span> <!-- Push the icon at the far right-->
+                                    <label for="hamburger">&#9776;</label>
+                                    <input type="checkbox" id="hamburger"/>
+
+                                <!-- (C) MENU ITEMS -->
+                                <div class="hamitems">
+                                    <a href="#">HOME</a>
+                                    <a href="#">CAKES</a>
+                                    <a href="#">ORDERING</a>
+                                    <a href="#">LESSONS</a>
+                                    <a href="#">ABOUT US</a>
+                                </div>
+                            </nav>    
+                        </section>   
+
+                        <section class="mobile-heading">
+                             <h1 class="mainheading"><em>ASTERISK ET OBELIX<br/>CAKES!</em></h1>
+                            <br/>
+                            <div class="thequote">
+                                <p><em>We Just Do Great Cakes!<br/><br/>The best cakes in town delivered to your door</em></p>
+                            </div>
+                        </section>
+
+                        <!-- <section class="desktop-heading">
+                            <h1 class="mainheading2">ASTERISK ET OBELIX<br/>CAKES!</h1>
+                           <br/>
+                           <div class="thequote2">
+                               <p><em>We Just Do Great Cakes!<br/><br/>The best cakes in town delivered to your door</em></p>
+                           </div>
+                        </section> -->
```

Comment:
There's a lot of commented out code in here. Avoid this. 
Comments to explain something or make a mental note is fine - but not commenting out actual code that you've decided not to use.